rust/operator-binary/src/main.rs

```diff
@@ -39,7 +39,11 @@ struct Opts {
 
 #[tokio::main]
 async fn main() -> anyhow::Result<()> {
-    stackable_operator::logging::initialize_logging("SUPERSET_OPERATOR_LOG");
+    stackable_operator::logging::initialize_logging(
+        "SUPERSET_OPERATOR_LOG",
+        APP_NAME,
+        TracingTarget::None,
+    );
```

Comment:
I think we must pass the tracingTarget through. Please have a look here, how i did it here https://github.com/stackabletech/commons-operator/blob/5dd7bf1f2acaea4121276c423a7d203700428b9b/rust/operator-binary/src/main.rs#L25-L42

Reply:
Tracing target forwarded